lisa/trace.py

```diff
@@ -3921,10 +3921,10 @@ def df_event(self, event, raw=None, window=None, signals=None, signals_init=True
 
         # Ensure the event name is set, as this sort of metadata might not be
         # saved in the swap
-        df.name = event
+        df.event_name = event
         # This is how it should be done. Keep the previous line in case someone
         # depends on it.
-        df.attrs['name'] = event
```

Comment:
that's actually not an issue, attrs is a dict that is not conflicting in any way with the actual dataframe columns (that's probably one of the reason it was introduced):
https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.attrs.html